### ISSUE TITLE
cannot divide by zero with bezier

### ISSUE THREAD

**Issue by scrouthtv** [johannes-wolf/typst-canvas#166]
Hello,

I am currently encountering an issue with some bezier curves, where compilation fails with `error: cannot divide by zero`. e.g.

```
#import "@preview/cetz:0.1.0"

#cetz.canvas({
    import cetz.draw: *

    bezier((1.5, 1.9), (1.8, 0.6), (1.8, 1.7))
})
```
Full error:
<details>

```
error: cannot divide by zero
    ┌─ @preview/cetz:0.1.0\src\bezier.typ:207:18
    │
207 │             ts.push((s * calc.sqrt(r) + a - 2 * c1 + c2) /
    │ ╭───────────────────^
208 │ │                   (a - b - 3 * c1 + 3 * c2))
    │ ╰───────────────────────────────────────────^

help: error occurred in this call of function `dim-extrema`
    ┌─ @preview/cetz:0.1.0\src\bezier.typ:219:13
    │
219 │       let ts = dim-extrema(s.at(dim, default: 0), e.at(dim, default: 0),
    │ ╭──────────────^
220 │ │                          c1.at(dim, default: 0), c2.at(dim, default: 0))
    │ ╰────────────────────────────────────────────────────────────────────────^

help: error occurred in this call of function `cubic-extrema`
   ┌─ @preview/cetz:0.1.0\src\path-util.typ:43:16
   │
43 │         bounds += bezier.cubic-extrema(
   │ ╭─────────────────^
44 │ │         s.at(1), s.at(2), s.at(3), s.at(4))
   │ ╰───────────────────────────────────────────^

help: error occurred in this call of function `bounds`
    ┌─ @preview/cetz:0.1.0\src\canvas.typ:134:26
    │
134 │         drawable.bounds = path-util.bounds(drawable.segments)
    │                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `process-element`
    ┌─ @preview/cetz:0.1.0\src\canvas.typ:309:12
    │
309 │     let r = process-element(element, ctx)
    │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

</details>

I usually fix it by slightly altering either coordinate, but in the long run, this might be a problem

**Comment by scrouthtv** [comment]
Another example: `bezier((1.5, 1.8), (1.9, 0.7), (1.8, 1.7))`

**Comment by johannes-wolf** [comment]
This has been fixed on master but not yet in a release.

**Comment by johannes-wolf** [comment]
Fixed in 0.1.1: https://github.com/typst/packages/pull/119
